checks/raw/signed_releases.go

```diff
@@ -15,15 +15,17 @@
 package raw
 
 import (
+	"errors"
 	"fmt"
 
 	"github.com/ossf/scorecard/v5/checker"
+	"github.com/ossf/scorecard/v5/clients"
 )
 
 // SignedReleases checks for presence of signed release check.
 func SignedReleases(c *checker.CheckRequest) (checker.SignedReleasesData, error) {
 	releases, err := c.RepoClient.ListReleases()
-	if err != nil {
+	if err != nil && !errors.Is(err, clients.ErrUnsupportedFeature) {
```

Comment:
release data is critical to the SignedReleases check, thoughts on keeping it?

Reply:
I'm not as familiar with the policy file, but I believe so, since it iterates over checks in the file
https://github.com/ossf/scorecard/blob/cb65b9b772049cb943b66c215902a5b86d216f05/policy/policy.go#L164-L170

Reply:
Okay, I'll revert this change then.

Reply:
I just double checked, and even with `mode: disabled` the `Signed-Releases` check fails with:

```
Error: check runtime error: Signed-Releases: internal error: unsupported feature
2025/06/27 10:25:30 error during command execution: check runtime error: Signed-Releases: internal error: unsupported feature
```

I need to completely omit it from `policy.yml` to get Scorecard to pass. I think it is valid to exclude it, given Azure DevOps doesn't support the Scorecard concept of a 'release'.